src/elements/hero/src/index.tsx

```diff
@@ -49,13 +53,26 @@ const Hero: React.FC<Props> = ({
     breadcrumbs &&
     React.cloneElement(breadcrumbs, { variant: breadcrumbVariant })
 
+  const getDeviceDisplay = () => {
+    if (fgImageOnMobile && fgImageOnTablet) {
+      return 'block'
+    } else if (!fgImageOnMobile && fgImageOnTablet) {
+      return ['none', 'block']
+    } else if (fgImageOnMobile && !fgImageOnTablet) {
+      return ['block', 'none', 'block']
+    }
+
+    return ['none', 'none', 'block']
+  }
+
   return (
     <div>
       <div
         sx={{
           position: 'relative',
           overflow: 'hidden',
-          variant: 'elements.hero.wrapper'
+          variant: 'elements.hero.wrapper',
+          ...(customBgColor && { backgroundColor: customBgColor })
```

Comment:
if we don't set a default value for `customBgColor` (L28)
then we can do this which is a little simpler, what do you think?

```suggestion
          backgroundColor: customBgColor
```